libs/sdk-bindings/src/breez_sdk.udl

```diff
@@ -496,7 +496,22 @@ enum BuyBitcoinProvider {
     "Moonpay",
 };
 
-interface BlockingBreezServices {   
+dictionary SweepRequest {
```

Comment:
A big weird we have Sweep & Withdraw (different names for the same functionality).
If we already have Sweep maybe we can change to:
PrepareSweepRequest/PrepareSweepResponse ?

Reply:
true; I haven't thought about that. changing names here

Reply:
Isn't it better to change the sweep to withdraw? So we keep the naming greenlight is using? @roeierez

Reply:
Don't have any preference. Perhaps if we already have sweep we can align wtih that. (PrepareSweep)